doc/source/examining/loading_data.rst

```diff
@@ -503,14 +503,22 @@ To load Enzo-P data into yt, provide the block list file:
    import yt
    ds = yt.load("hello-0200/hello-0200.block_list")
 
-Mesh fields are fully supported for 1, 2, and 3D datasets.
+Mesh and particle fields are fully supported for 1, 2, and 3D datasets.  Enzo-P
+supports arbitrary particle types defined by the user.  The available particle
+types will known as soon as the dataset index is created.
+
+.. code-block:: python
+
+   ds = yt.load("collapse-3d-0000/collapse-3d-0000.block_list")
+   ds.index
+   print (ds.particle_types)
```

Comment:
Are you sure you want to keep the whitespace between `print` and `(`?
In Python 3, `print()` is a function.

Reply:
Python allows spaces in between functions and the calling parentheses.  I just did it this way because it exists throughout yt like this.